tests/utils.py

```diff
@@ -413,7 +412,6 @@ def post_send(agr_obj, send_wr, gid_index, port, qp_idx=0):
     """
     qp_type = agr_obj.qp.qp_type
     if qp_type == e.IBV_QPT_UD:
-        ah = get_global_ah(agr_obj, gid_index, port)
```

Comment:
removing this will break current direct post_send calls on UD QPs such as in test_odp_ud_traffic (in test_odp). please add AH in such tests (I think it's the only one)

Reply:
Missed that, will fix